### PR TITLE
JR2-127 Add Candidate Detail component, route, service

### PR DESCRIPTION
**Pull request by Invizible**

### CODE REVIEW

File: src/main/webapp/app/candidate-search/services/candidate-profile.service.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@angular/core';
```
Comment (Invizible) [inline src/main/webapp/app/candidate-search/services/candidate-profile.service.ts]: wiil remove this service

File: src/main/webapp/app/candidate-search/candidate-detail/candidate-detail.component.html [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            <div class="job-detail__title">
+                <h2 class="text--light">Title</h2>
+            </div>
+
+            <div class="candidate-detail__content">
+                <span class="badge badge-blue">
+                    {{ 'global.reference.canton.' + candidateProfile.residenceCantonCode | translate}}
+                </span>
+                <span class="badge badge-gray">{{ candidateProfile.workLoad }}%</span>
+                <span class="badge badge-green">Anstellungsdauer</span>
```
Comment (varusakov) [inline src/main/webapp/app/candidate-search/candidate-detail/candidate-detail.component.html]: This property is missing in candidate model?

Comment (Invizible) [reply src/main/webapp/app/candidate-search/candidate-detail/candidate-detail.component.html]: yes

File: package.json [hunk 3]
```diff
@@ -30,7 +30,7 @@
     "swagger-ui": "2.2.10",
     "tether": "1.4.0",
     "zone.js": "0.8.16",
-    "@alv-ch/alv-styleguide": "0.0.8",
+    "@alv-ch/alv-styleguide": "0.0.9",
```
Comment (birom) [inline package.json]: Does the version 0.0.9 work now without any problems?

File: src/main/webapp/i18n/translations.csv [hunk 4]
```diff
@@ -233,6 +257,14 @@ global,global.reference.canton.VD,Waadt,Vaud,Vaud,Waadt
 global,global.reference.canton.VS,Wallis,Valais,Vallese,Wallis
 global,global.reference.canton.ZG,Zug,Zoug,Zugo,Zug
 global,global.reference.canton.ZH,Zürich,Zurich,Zurigo,Zürich
+global,global.reference.languages,Sprachkenntnisse,Sprachkenntnisse,Sprachen,Sprachen
+global,global.reference.language,Sprache,Sprache,Sprache,Sprache
```
Comment (birom) [inline src/main/webapp/i18n/translations.csv]: Is this key used somewhere?

Comment (Invizible) [reply src/main/webapp/i18n/translations.csv]: no

Comment (birom) [reply src/main/webapp/i18n/translations.csv]: Could you please remove it and regenerate the translation json files.
Thank you!

File: package.json (hunk 3 above)
Comment (Invizible) [reply package.json]: Actually I saw only one problem and fixed it

**Pull request merged**